nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -626,14 +620,21 @@ $(XamlMetaDataProviderPch)
         </PropertyGroup>
     </Target>
 
-    <!-- Fast ABI component support -->
-    <PropertyGroup Condition="'$(CppWinRTFastAbi)'=='true'">
-        <CppWinRTParameters>$(CppWinRTParameters) -fastabi</CppWinRTParameters>
-    </PropertyGroup>
-    <ItemDefinitionGroup Condition="'$(CppWinRTFastAbi)'=='true'">
-        <Link>
-            <AdditionalDependencies>%(AdditionalDependencies);$(CppWinRTPackageDir)build\native\lib\$(Platform)\cppwinrt_fast_forwarder.lib</AdditionalDependencies>
-        </Link>
+    <!--Append any additional item metadata after all default and project settings have been applied-->
+    <ItemDefinitionGroup>
+      <ClCompile>
+        <AdditionalOptions>%(AdditionalOptions) /bigobj /await</AdditionalOptions>
```

Comment:
I don't think we should force the use of await?

Reply:
This isn't new (just relocated) - without it, async projection headers fail.

Reply:
Same applies to /std:c++17 though, which we don't add here either.

Reply:
user could add /std::latest